### PR TITLE
[ENG-1155] Add wiki dump for IA prototype

### PR DESCRIPTION
**Pull request by Johnetordoff**
⚠️ Don't use Docker for this! ⚠️ 
⚠️ This will Inevitably fail CI because it's PY3, review anyway!  ⚠️ 

## Purpose

Dump wiki markdown files  to local directory.

## Changes

- add little script for dumping and writing to file.

## QA Notes

Ok devs to test this you're going to have to dust off your trusty old virtualenv, this runs specifically in python3 so you're going to have specify that on  creation with something like ` mkvirtualenv IA --python=python3` then `pip install -r scripts/IA/requirements.txt`  that should cover dependencies. 

## Documentation

Not user facing

## Side Effects

None that I know of.
 
## Ticket

https://openscience.atlassian.net/browse/ENG-1155

### CODE REVIEW

File: IA/dump_wiki.py [hunk 1]
```diff
@@ -0,0 +1,77 @@
...
+    write_tasks = []
+    for page in pages_as_list:
+        write_tasks.append(write_wiki_content(page))
+
+    await asyncio.gather(*write_tasks)
+
+if __name__ == '__main__':
+    parser = argparse.ArgumentParser()
+    parser.add_argument('-id', '--guid', help='The guid of the registration if who\'s wiki you want to dump.')
+    args = parser.parse_args()
```
[the remark this answers is not in the document]
Comment (Johnetordoff) [reply IA/dump_wiki.py]: @corbinSanders not sure what you mean, like additional validation?

[the remark this answers is not in the document]
Comment (corbinSanders) [reply IA/dump_wiki.py]: If you run this without a guid, it errors out on the get request. Perhaps adding a little valueerror if a guid isn't input. Just for cleanliness. Again, "Would be nice to have"

**Comment by corbinSanders** [comment]
LGTM!

File: IA/dump_wiki.py [hunk 2]
```diff
@@ -0,0 +1,90 @@
...
+
+TEN_SECONDS = 10
+
+
+class WikiDumpError(Exception):
+    pass
+
+
+@sleep_and_retry
+@limits(calls=50, period=TEN_SECONDS)
```
[the remark this answers is not in the document]
Comment (Johnetordoff) [reply IA/dump_wiki.py]: Yes, but I don't think we actually have that rate limiting behavior set up for the API, do we?

[the remark this answers is not in the document]
Comment (brianjgeiger) [reply IA/dump_wiki.py]: Right now, the v2 API will give you a 429 and tell you the time to wait for a specific amount of time if you make too many requests. It's not working for waterbutler, but it is for OSF.

**Pull request closed without merging**